kalite/store/tests.py

```diff
@@ -38,73 +38,12 @@ class SwitchTest(BaseTest):
-        that a gift_card is created with appropriate points.
-        """
-        set_current_unit_settings_value(self.facility.id, self.unit)  # make sure the unit and grade we are in match the test
-        test_object = Test.all().get(self.exam_id, None)
-        self.assertTrue(test_object)
-        testlog = TestLog(user=self.student, test=self.exam_id, index=20, started=True, complete=True, total_correct=15, total_number=21)
-        testlog.save()
-        set_current_unit_settings_value(self.facility.id, self.unit)
-        # Set and unset exam mode to trigger signal.
-        set_exam_mode_on(test_object)
-        set_exam_mode_on(test_object)
-        transactionlogs = StoreTransactionLog.objects.filter(user=self.student, context_id=str(self.unit), context_type="output_condition", item="gift_card")
-        self.assertTrue(len(transactionlogs))
-        for transactionlog in transactionlogs:
-            self.assertTrue(transactionlog.value == int(round(settings.UNIT_POINTS*float(testlog.total_correct)/testlog.total_number)))
-
-
-    @override_settings(CONFIG_PACKAGE=["Nalanda"])
-    def test_unit_switch(self):
-        """
-        Tests that when a unit is switched a negative value gift_card
-        is created to nullify current user points.
-        """
-        set_current_unit_settings_value(self.facility.id, 2)
-
-        exerciselog = ExerciseLog(user=self.student, exercise_id="addition_1", streak_progress=8, attempts=8, points=10, complete=True)
-        exerciselog.save()
-
-        # Set to unit 3 to trigger signal.
-        set_current_unit_settings_value(self.facility.id, 3)
-
-        transactionlogs = StoreTransactionLog.objects.filter(user=self.student, context_id="2", context_type="unit_points_reset", item="gift_card")
-        self.assertTrue(len(transactionlogs)==1)
-
-        for transactionlog in transactionlogs:
-            self.assertTrue(transactionlog.value==-exerciselog.points)
-
-        newexerciselog = ExerciseLog(user=self.student, exercise_id="addition_2", streak_progress=8, attempts=8, points=20, complete=True)
-        newexerciselog.save()
-
-        # Move back a unit
-        set_current_unit_settings_value(self.facility.id, 2)
-
-        # Check that newexerciselog has been nullified
-        transactionlogs = StoreTransactionLog.objects.filter(user=self.student, context_id="3", context_type="unit_points_reset", item="gift_card")
-        self.assertTrue(len(transactionlogs)==1)
-        for transactionlog in transactionlogs:
-            self.assertTrue(transactionlog.value==-newexerciselog.points)
-
-        # Check that the original transactionlog has been deleted
-        transactionlogs = StoreTransactionLog.objects.filter(user=self.student, context_id="2", context_type="unit_points_reset", item="gift_card")
-        self.assertTrue(len(transactionlogs)==0)
-
-    @override_settings(CONFIG_PACKAGE=["Nalanda"])
     def test_playlist_group_mapping_reset_for_a_facility(self):
```

Comment:
Again, this functionality doesn't get called because of no unit-switching, so we probably shouldn't test it.